### DISCUSSION TITLE
Welcome to Xbox Controller button remapper Discussions!

### DISCUSSION THREAD

**Discussion by Adam777Z** [Adam777Z/xbox-controller-button-remapper#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by KreeperSteve** [comment]
I'm having trouble getting it working in Skyrim SE

**Reply by Adam777Z** [reply]
[Help is available here](https://github.com/Adam777Z/xbox-controller-button-remapper/discussions/2).

**Comment by AlexFDR** [comment]
Would it be possible to get this to work with key combinations? As in, for example, Share+LB = media key play/pause, and stuff like that.

I recently got an xbs controller to replace my ps4 controller and one of the things i'm missing the most is that ds4win allowed me to use the touch pad for media controls which i used a lot. Thanks.

**Reply by Adam777Z** [reply]
This is currently not supported. As an alternative, you can try setting `share_hold_mode = 2` and `share_longpress_key` and hold down the Share button for the configured duration to send the key(s).

**Comment by TRRAXT** [comment]
Would it be possible to add double click recognise on share and Xbox buttons?

**Reply by Adam777Z** [reply]
Currently only keyboard keys are supported.
